packages/ember-data/lib/system/adapter.js

```diff
@@ -199,13 +199,16 @@ var Adapter = Ember.Object.extend({
 
     Example
 
-    ```javascript
-    App.ApplicationAdapter = DS.Adapter.extend({
-      queryRecord: function(store, typeClass, query) {
-        var url = [type.typeKey, id].join('/');
+    ```app/adapters/application.js
+    import DS from 'ember-data';
+    import Ember from 'ember';
+
+    export default DS.Adapter.extend(DS.BuildURLMixin, {
+      queryRecord: function(store, type, query) {
+        var urlForQueryRecord = this.buildURL(type.modelName, null, null, 'queryRecord', query);
```

Comment:
Just for the record I didn't mean to change the variable name but to call urlForQueryRecord instead of buildURL :)